src/components/FeedbackAnalyticsInfo.tsx

```diff
@@ -1,70 +0,0 @@
-import { FC } from "react"
-import {
-  HStack,
-  Image,
-  LabelSm,
-  BodySm,
-  Stack,
-  useColorModeValue,
-  StackProps,
-} from "@threshold-network/components"
-import sunglassesLight from "../static/images/SunglassesLight.png"
-import sunglassesDark from "../static/images/SunglassesDark.png"
-import diagonalPillLight from "../static/images/DiagonalPillLight.png"
-import diagonalPillDark from "../static/images/DiagonalPillDark.png"
-import diagonalArrowLight from "../static/images/DiagonalArrowLight.png"
-import diagonalArrowDark from "../static/images/DiagonalArrowDark.png"
```

Comment:
Let's also remove those images from the repo

Reply:
Ref commit: 080cff06